fe/fe-core/src/main/java/org/apache/doris/nereids/processor/post/RecomputeOutputProcessor.java

```diff
@@ -22,13 +22,14 @@
 import org.apache.doris.nereids.trees.plans.physical.AbstractPhysicalPlan;
 
 /**
- * merge consecutive projects
+ * recompute output in logical properties
+ * after join reorder, plan output slot order is changed, recompute them.
  */
-public class RecomputeLogicalPropertiesProcessor extends PlanPostProcessor {
+public class RecomputeOutputProcessor extends PlanPostProcessor {
```

Comment:
why rename this file? RecomputeLogicalProperties is more accurate

Reply:
LogicalProperties contains 2 parts: output and dataTrait
the purpose of this rule is to reset output, not dataTrait.
when we implement this rule, dataTrait is not a part of logicalProperties.